SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialRPCService.h

```diff
@@ -44,21 +47,31 @@ struct EntityRPCType
 
 enum class EPushRPCResult : uint8
 {
-	Success,
-
-	QueueOverflowed,
-	DropOverflowed,
-	HasAckAuthority,
-	NoRingBufferAuthority
+	None = 0x00,
+	// First group of flags tracks the Outcome (success/failure reason) of an attempt to push an RPC
+	Success = 0x01,
```

Comment:
As the error cases available also dependent on if we're attempting to send or recieve the RPC, do you think we could incorporate that information here somehow? It's a little difficult reasoning about all the ways an RPC may get stalled currently.

Reply:
This PR is strictly dealing with sending an RPC - receiving is handled separately and doesn't use EPushRPCResult.

Reply:
I guess because the RPCService deals with both, I find it a little overlapping. Could `SetRPCFailureOutcomesToQueue` reference that this is for sending rpcs only?

Reply:
Definitely

Reply:
Function and internal variable have been renamed to indicate they are for push RPCs, and a comment has been added to the declaration indicating sender only.